wdl-lint/src/rules/inconsistent_newlines.rs

```diff
@@ -22,7 +22,7 @@ fn inconsistent_newlines(span: Span) -> Diagnostic {
     Diagnostic::note("inconsistent newlines detected")
         .with_rule(ID)
         .with_label("the first occurrence of a mismatched newline is here", span)
-        .with_fix("use either \"\\n\" or \"\\r\\n\" consistently in the file")
+        .with_fix("ensure that all newlines are consistent")
```

Comment:
```suggestion
        .with_fix("ensure that the same newlines (e.g., \"\\n\" or \"\\r\\n\") are used")
```

We probably need to include the literal characters still just to make it more readily apparent what the problem is here. It may take some extra mental parsing to understand the issue unless you list the two items.